src/dotnet-interactive.Tests/FileProviderTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using Microsoft.DotNet.Interactive.App.Http;
+using Microsoft.DotNet.Interactive.Tests;
+using Microsoft.DotNet.Interactive.Tests.Utility;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Interactive.App.Tests
```

Comment:
To the extent you're testing interfaces directly and not over HTTP, it looks like these belong in `Microsoft.DotNet.Interactive.Tests`.

Reply:
that could bring too many asp.net dependencies